app/src/main/java/fr/neamar/kiss/result/Result.java

```diff
@@ -202,4 +216,14 @@ public int getThemeFillColor(Context context) {
         TypedArray ta = context.obtainStyledAttributes(attrs);
         return ta.getColor(0, Color.WHITE);
     }
+
+    @TargetApi(Build.VERSION_CODES.HONEYCOMB)
+    protected void inflateBaseMenu(Context context, PopupMenu menu) {
+        //if you have selected to use favorites instead of history then show additional menus
+        if (PreferenceManager.getDefaultSharedPreferences(context).getBoolean("replace-app-history-with-favs", false)) {
+            if (context instanceof MainActivity) {
+                menu.getMenuInflater().inflate(R.menu.menu_item_favorite_add, menu.getMenu());
```

Comment:
since we're doing this every time, it may be simpler to simply add the "Favorties" button to the main menu and remove this function altogether.